src/libtorchaudio/forced_align/cpu/compute.cpp

```diff
@@ -202,9 +193,14 @@ std::tuple<Tensor, Tensor> compute(
       "forced_align_impl",
       AT_WRAP([&] {
         if (targets.scalar_type() == ScalarType::Long) {
-          forced_align_long_impl<scalar_t>(logProbs, targets, blank, paths);
+          (forced_align_impl<scalar_t, int64_t>(
+              logProbs, targets, blank, paths));
+        } else if (targets.scalar_type() == ScalarType::Int) {
+          (forced_align_impl<scalar_t, int32_t>(
```

Comment:
This change fixes access violation in one of the autograd tests on Windows CPU.
It is likely related to MSVC compiler handling of templates that sometimes have mysterious (d)effects.